kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/tables/LessonLearnersTable.vue

```diff
@@ -0,0 +1,98 @@
+<template>
```

Comment:
I was wondering where can live these Tables components, and Im planning in having them in a `/tables` folder of the page that uses them if its used in just one tab, but for tables that are used accross multiple tabs, have them in `/common/tables` rather than in a `reports` folder.

Reply:
That seems fine to me.

Reply:
Are these completely new components or copies? Might be good to make sure that git/github recognizes them as moves so that we retain the blame (and you don't take all of it!)

Reply:
No, they are copies, because I havent removed the reports pages that also use these components yet and I just didnt want to mess updating the path for these components that will be removed anyways.